src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs

```diff
@@ -398,9 +396,7 @@ private string GetXmlValue(XPathNavigator node)
             // <remarks><para>Value</para></remarks>
             // decode InnerXml as it encodes
             var lineInfo = node as IXmlLineInfo;
-            var lineNumber = lineInfo.LineNumber - 1;
-            var line = _lines[lineNumber];
-            int column = GetNonWhitespaceIndex(line);
+            int column = lineInfo.HasLineInfo() ? lineInfo.LinePosition - 2 : 0;
```

Comment:
LinePosition starts from 1 and it would ignore '<'(e.g. \<summary\> the LinePosition is the column number of 's'), so i need to -2